e2e/cypress/tests/integration/channel_sidebar/dm_gm_filtering_sorting_spec.js

```diff
@@ -56,14 +56,14 @@ describe('DM/GM filtering and sorting', () => {
         // # Expand the DM category (so that we can check all unread DMs quickly without the sidebar scrolling being an issue)
         cy.get('button.SidebarChannelGroupHeader_groupButton:contains(DIRECT MESSAGES)').should('be.visible').click();
 
-        // * Verify that there are 20 DMs shown in the sidebar
```

Comment:
I would add the ticket number here just to refer back on in future on why it is 40.